### DISCUSSION TITLE
Mac M1

### DISCUSSION THREAD

**Discussion by max3ndeavour** [alzayats/DeepFish#11] [General]
Hi, 

I was interested in your project but the architecture was incompatible to Mac M1. I made it work locally on my Mac. If you're interested, we can discuss an update to insert those changes in your project. 

Regards

Max

**Comment by alzayats** [comment]
Hey Max,

Thanks for reaching out! It’s great to hear that you were able to make our project work on your Mac M1. We’d love to see the changes you made. If you’re up for it, feel free to submit a pull request to update the code.  

Cheers, 
ALzayat
